src/lib/components/Card/Card.tsx

```diff
@@ -57,14 +59,36 @@ export const Card: FC<CardProps> = ({
       )}
       {...theirProps}
     >
-      {imgSrc && (
-        <img
-          alt={imgAlt ?? ''}
-          src={imgSrc}
-          className={classNames(theme.img.base, theme.img.horizontal[horizontal ? 'on' : 'off'])}
-        />
-      )}
+      <Image {...props} />
       <div className={theme.root.children}>{children}</div>
     </Component>
   );
 };
+
+const Image: FC<CardProps> = ({ theme: customTheme = {}, ...props }) => {
+  const theme = mergeDeep(useTheme().theme.card, customTheme);
+  if (props.renderImage) {
+    return props.renderImage();
```

Comment:
We need to find a way to properly pass the theme classes to the rendered image. Otherwise, this tag will not be properly styled to flowbite.

Reply:
The flowbite-react theme system allows you to centralize the style of the components in one single place. So, it means that all your Cards will first use the style from the default theme unless you decide to overwrite it. It is necessary from the library perspective that all things inside the components should be accessible by the theme properties to be stylized. It is a way that we found to keep the design system consistency, but it also allows exceptions (when really necessary).

Reply:
It is an image. What do you want to style with an image?

Reply:
[Well, first that we already ](https://github.com/themesberg/flowbite-react/blob/89d58dc03727d150e73ec753748eb98640bc84a3/src/lib/theme/default.ts#L332)

☝️ Cards can be horizontal or vertical layouts, and it impacts how the image will be styled.

Reply:
Maybe the "custom image component" is a `div` which gets a background image. Maybe someone wants to render a `form`. Who knows what kind of hacky optimization magic the user might be doing. We cannot foresee what they come up with. Better tell them: You are on your own, no magic provided on our end.

Reply:
I think this discussion is overlapping with the one that we are having in the other component. Let's keep it there... I'm happy with `renderImage` property.